### ISSUE TITLE
Deactivate / Hide navbar icons on custom landing page

### ISSUE THREAD

**Issue by kevinrue** [iSEE/iSEE#597]
**Is your feature request related to a problem? Please describe.**

Most observers are not defined yet when the landing page is shown, leading to a confusing situation where users can click on navbar menu items that don't do anything until the main app is launched.

That said, https://github.com/kevinrue/iSEEExperimentHub defines a landing page where I defined an observer for the tour button as soon as the landing page is defined, giving a tour to the landing page (when the main app is started, the button then launches the tour for the main app, if defined).

**Describe the solution you'd like**

Some of the navbar icons do not apply to the landing page (e.g., extract the R code) and could be hidden altogether.
Others (like the tour mentioned above) have the potential to be overridden for the purpose of the landing page.

I don't have a clear plan in mind, and I'm not even sure how flexible the navbar is form that perspective, but it would be nice to hide/disable buttons 

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by kevinrue** [comment]
See https://github.com/kevinrue/iSEEhub/pull/12

@LTLA 
You'll note that `iSEEhub` uses `:::` to access the identifier of those buttons.
This is obviously sub-optimal, but so would be referring to their name as a character string that might change in `iSEE` (unlikely, but less clean for traceability too)

Thoughts?

**Comment by LTLA** [comment]
Seems like these buttons should be disabled by the core landing page machinery in **iSEE** itself.

**Comment by kevinrue** [comment]
That is true, but as demonstrated in `iSEEhub`, some of them can be given a purpose in landing pages, which means that their identifier should be exported by `iSEE` so that scripts and packages that design landing pages can cleanly access those identifiers without relying on `:::`.
